### ISSUE TITLE
BULGARIA

### ISSUE THREAD

**Issue by asfi30** [electricitymaps/electricitymaps-contrib#4809]
Hi,

Bulgaria Installed capacities are as follows:
nuclear: 2000 MW
geothermal: 0 MW
biomass: 79 MW
coal: 4475 MW
wind: 705 MW
solar: 1246 MW
hydro: 3213 MW
hydro storage: 931 MW
battery storage 0MW
gas: 1269 MW
oil 0 MW
unknown 0MW

This information can be checked from the following link https://www.eso.bg/fileObj.php?oid=3572 This is the website of the Bulgarian Electricity System Operator, 

In case you decide to update the information.

Best regards

**Comment by asfi30** [comment]
Hi
 
  This is the English version
https://www.eso.bg/fileObj.php?oid=3573
  In Bulgaria batteries are currently only for self usage purposes of households or Small business
 and not associated to the grid or to the trading market on IBEX (https://ibex.bg/ ).
 The national resilience plan plans to implement batteries 2024 and onward. It is obligatory for the Electricity System Operator (ESO) to
 report on annual basis all associated to the grid electricity manufacturers. If there is no information for "unknown, batteries or oil" it means they are not associated to the national grid. Small companies do have UPS systems or small diesel generator, but they are used as back up only and not allowed to transmit towards the grid.
 Please keep in mind ESO report concerns year 2021. The 2022 report will be available Q2 or Q3 2023.
  
  Kind regards,  Aleks Latev









 >-------- Оригинално писмо --------

 >От: Viktor Andersson ***@***.***

 >Относно: Re: [electricitymaps/electricitymaps-contrib] BULGARIA (Issue #4809)

 >До: electricitymaps/electricitymaps-contrib  

 >Изпратено на: 28.11.2022 12:04



 
 
  
  
  
Does it explicitly say that battery storage and unknown is 0 MW?
 I find it doubtful that battery storage is actually 0 since most countries have at least a few grid-connected batteries by now, either residential or utility (even if they aren't very big).

**Comment by VIKTORVAV99** [comment]
Okay then I think we can add battery and oil as 0 but leave unknown as is (as it is per definition unknown).

I'll add the help wanted and good first issue labels if anyone wants to do it, otherwise I'll do it myself in a few days.

**Comment by calucido** [comment]
Hi, I'll take a shot at this!

ENTSO-E has 2022 capacity data for Bulgaria, but both ENTSO-E and ESO appear to under-report nuclear capacity: nuclear production at 21:00UTC today is 2160MW, while the reported capacities are 2080MW and 2000MW, respectively.

Is this an acceptable margin of error given the information that we have?

**Comment by VIKTORVAV99** [comment]
> Is this an acceptable margin of error given the information that we have?

It is, sometimes it can take a while for the official capacities to update everywhere even when the technical capacity is higher.

Would you like me to assign you the issue?

**Comment by asfi30** [comment]
Hi, 
  Installed Capacity of WWER ( water-water energetic reactor ) of Nuclear power plant is 2*1000MW in Bulgaria (3rd Generation V-320). Both ENTSO-E and ESO report like this and same on the NPP Kozloduy (https://www.kznpp.org/ ) under the REMIT program. It is just one NPP.
 However those type of reactors can achieve more than 100% of installed capacities. In fact up to 110% by increasing the supplied thermal power. Block 5 is curently runing at 1080MW and block 6 at 1082MW. SO the value of 2160
 reported by ENTSO-E/ESO is correct, which can be seen on the NPP website as well.
 Hope this makes sense.
 

  





 >-------- Оригинално писмо --------

 >От: calucido ***@***.***

 >Относно: Re: [electricitymaps/electricitymaps-contrib] Bulgaria capacities
 (Issue #4809)

 >До: electricitymaps/electricitymaps-contrib  

 >Изпратено на: 02.12.2022 01:14



 
 
  
  
  
Hi, I'll take a shot at this!  
  
ENTSO-E has 2022 capacity data for Bulgaria, but both ENTSO-E and ESO appear to under-report nuclear capacity: nuclear production at 21:00UTC today is 2160MW, while the reported capacities are 2080MW and 2000MW, respectively.  
  
Is this an acceptable margin of error given the information that we have?

**Comment by calucido** [comment]
Yes, please assign it to me. Since the reactors seem to be exceeding their "installed capacities", I'll just use the ENTSO-E 2022 installed capacities, since that seems to be what matters for this feature.

**Comment by VIKTORVAV99** [comment]
> Yes, please assign it to me.

There we go. 🙂

**Comment by asfi30** [comment]
Hi, 
  This is the link from ENTSO-E for 2020
 https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show?name=&amp;defaultValue=true&amp;viewType=TABLE&amp;areaType=BZN&amp;atch=false&amp;dateTime.dateTime=01.01.2022+00:00|UTC|YEAR&amp;dateTime.endDateTime=01.01.2022+00:00|UTC|YEAR&amp;area.values=CTY|10YCA-BULGARIA-R!BZN|10YCA-BULGARIA-R&amp;productionType.values=B01&amp;productionType.values=B02&amp;productionType.values=B03&amp;productionType.values=B04&amp;productionType.values=B05&amp;productionType.values=B06&amp;productionType.values=B07&amp;productionType.values=B08&amp;productionType.values=B09&amp;productionType.values=B10&amp;productionType.values=B11&amp;productionType.values=B12&amp;productionType.values=B13&amp;productionType.values=B14&amp;productionType.values=B20&amp;productionType.values=B15&amp;productionType.values=B16&amp;productionType.values=B17&amp;productionType.values=B18&amp;productionType.values=B19

  
  Still not appearing the right data for installed capacities on Electricity Maps (EM)  
  Nuclear 2080MW (EM is 2000)  
Geothermal 0/0  correct   Biomass 81 MW(EM is 80)  Coal 4475 MW (EM is 4370)  Wind 705 MW (EM is 701)  Solar
 1313 MW (EM is 1130)  Hydro 2349 MW (EM is 2740)  Hydro Storage 864 MW (EM is 931)  Battery storage 0 MW (EM is NA)  Gas 1266 MW (EM is 1370 MW)  Oil 0/0  correct   unknown NA - correct








 >-------- Оригинално писмо --------

 >От: Viktor Andersson ***@***.***

 >Относно: Re: [electricitymaps/electricitymaps-contrib] Bulgaria capacities
 (Issue #4809)

 >До: electricitymaps/electricitymaps-contrib  

 >Изпратено на: 06.12.2022 18:20



 
 
  
  
  
Closed  #4809  as completed via  #4832 .

**Comment by VIKTORVAV99** [comment]
The change won't be visible until a new build/version of the app is deployed.
That should be soon though.
